src/style.css

```
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  height: 15px;
+  display: flex;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  background: url("./images/star.svg") center no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__item:nth-child(-n+1),
+.stars--2 .stars__item:nth-child(-n+2),
+.stars--3 .stars__item:nth-child(-n+3),
+.stars--4 .stars__item:nth-child(-n+4),
+.stars--5 .stars__item:nth-child(-n+5) {
```

since there is only five stars
```suggestion
.stars--5 .stars__item {
```